.github/workflows/mysql.yml

```diff
@@ -25,9 +25,14 @@ jobs:
         # will assign a random free host port
         - 3306/tcp
         # needed because the container does not provide a healthcheck
-        options: --health-cmd "mysqladmin ping --silent" --health-interval 30s --health-timeout 30s --health-retries 10
+        options: >-
+          --health-cmd "mysqladmin ping --silent" --health-interval 30s --health-timeout 30s
+          --health-retries 10 -v /data/mysql:/var/lib/mysql
+
 
     steps:
+    - run: docker logs ${{ job.services.mysql.id }}
+    - run: ls /data/mysql
```

Comment:
You can remove these two lines if you want, I was just using them to debug some stuff.